egs/aishell/asr1/path.sh

```diff
@@ -2,7 +2,7 @@ MAIN_ROOT=$PWD/../../..
 KALDI_ROOT=$MAIN_ROOT/tools/kaldi
 
 [ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
-export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$KALDI_ROOT/tools/sctk/bin:$PWD:$PATH
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$KALDI_ROOT/tools/sctk/bin:$PWD:$MAIN_ROOT/tools/kenlm/build/bin:$PATH
```

Comment:
This script should check if `tools/kenlm.done` exists. If it doesn't, show error messages like https://github.com/espnet/espnet/blob/292a15e51dca33e0339fa324062e5c4b5b8f4924/egs/fisher_callhome_spanish/st1/path.sh#L21-L27